exercises/variable-length-quantity/.meta/template.j2

```diff
@@ -0,0 +1,36 @@
+{%- import "generator_macros.j2" as macros with context -%}
+
+{%- macro list_int_to_hex(integers) %}
+            [
+                {% for integer in integers -%}
+                {{ "0x{:x}".format(integer) }},
+                {% endfor %}
+            ]
+{% endmacro -%}
+
+{{ macros.header() }}
+
+class {{ exercise | camel_case }}Test(unittest.TestCase):
+    {% for case in cases -%}
+        {%- for sub_case in case.cases %}
+    def test_{{ sub_case["description"] | to_snake }}(self):
+        {%- if "input" not in sub_case %}
+        #
+        # {{ sub_case.keys() }}
+        #
+        pass
+        {%- else %}
```

Comment:
This appears to be debugging code. Please remove it, as it does not impact the final result.

Reply:
Sorry about that one